dolphin/processor/config.py

```diff
@@ -302,24 +313,56 @@ def custom_logL_addition(self, kwargs_lens=None, kwargs_source=None,
                              kwargs_special=None,
                              kwargs_extinction=None):
         """
-        Impose a Gaussian prior to limit the Maximum allowed position angle
+        Add different types of custom log L funtions
+        1) Impose a Gaussian prior to limit the Maximum allowed position angle
          difference between lens mass and lens light (in degrees).
+        2) Impose a Gaussian prior to limit the ratio between the lens mass and
```

Comment:
Please clarify what quantity of the lens mass and light this "ratio" correspond to.